README.md

```diff
@@ -1,44 +1,80 @@
 # Formal Ledger Specifications
 
-This repository contains the formal ledger specifications that are intended to eventually replace the existing formal specifications of the Cardano ledger found [here](https://github.com/IntersectMBO/cardano-ledger). This project is currently incomplete and work in progress.
+This repository contains the formal ledger specifications for the Cardano blockchain,
+written in [Agda](https://wiki.portal.chalmers.se/agda/pmwiki.php). These
+specifications are executable, and the project is a work-in-progress which will
+eventually replace the existing TeX-based formal specifications.
 
-This repository currently contains the work-in-progress specification for Cardano (up to and including the Conway era). The specification is executable and contains some documentation in the form of a PDF document. It can be built by following the steps in [Building](#building).
+The repository provides the specification for the Cardano ledger up to and including
+the Conway era. You can interact with the specification by building the artifacts,
+generating Haskell source code for conformance testing, or exploring the Agda source
+in a development environment.
 
-| Formal Specification | HTML Version |
-| -------------------- | ------------ |
-| [Full Cardano Ledger](https://IntersectMBO.github.io/formal-ledger-specifications/cardano-ledger.pdf) | [Ledger](https://IntersectMBO.github.io/formal-ledger-specifications/html/index.html) |
-| [Conway](https://IntersectMBO.github.io/formal-ledger-specifications/conway-ledger.pdf) | [Ledger](https://IntersectMBO.github.io/formal-ledger-specifications/html/index.html) |
+---
+
+## **Published Artifacts**
+
+The official PDF specifications and a browseable HTML version of the source code are generated by our CI pipeline and published.
+
+| Artifact | Link | Description |
+| :---- | :---- | :---- |
+| **Full PDF Specification** | [cardano-ledger.pdf](https://IntersectMBO.github.io/formal-ledger-specifications/cardano-ledger.pdf) | The complete specification for the Cardano Ledger. |
+| **Conway-Era PDF** | [conway-ledger.pdf](https://IntersectMBO.github.io/formal-ledger-specifications/conway-ledger.pdf) | A differential specification for the Conway era. |
+| **HTML Source Code** | [Browseable HTML](https://IntersectMBO.github.io/formal-ledger-specifications/html/index.html) | An interactive, hyperlinked version of the Agda source code. |
+
+**Note on PDFs:** As we are migrating from a LaTeX-based to a Markdown-based documentation workflow, **building** PDFs **from source is currently deprecated and unsupported.** Please use the links above to access the official, pre-built PDF documents.
 
 ---
 
-## Building
+## **🚀 Quick Start: Building Local Artifacts**
```

Comment:
I think we should remove the `building` section from the README. At most just explain how to explore interactively the mechanization.